hw/ip/otbn/doc/dv/index.md

```diff
@@ -155,75 +167,107 @@ This paragraph implies eight coverage points (four flags times two values) for t
 
 #### ADD
 
+This instruction uses the `R` encoding schema, with covergroup `enc_r_cg`.
+
 - Cross the three possible signs (negative, zero, positive) for each input operand (giving 9 points).
```

Comment:
nit: R-tytpe has 2 input and one output operands.

Reply:
Yep, and 3 squared is 9. I don't think I understand this comment?